tasks/add-fbc-contribution/README.md

```diff
@@ -14,6 +14,11 @@ Task to create a internalrequest to add fbc contributions to index images
 | pipelineRunUid | The uid of the current pipelineRun. Used as a label value when creating internal requests | No       |                      |
 | targetIndex    | targetIndex value updated by update-ocp-tag task                                          | No       |                      |
 
+## changes in 2.1.1
+- fix the error message for the empty value of `issueId`, `productName` and `productVersion`
+Comment
+  - with the old format, the backticks caused the string inside (e.g. fbc.issueId) to be executed as a command
```

Comment:
This is a continuation of line 18. So, can you remove the `-` at the start?

Reply:
Updated.

Reply:
When I suggested it, I meant it as a second level bullet point. And I don't think we normally split one bullet point into multiple lines like this. But I'm fine with it.

Reply:
The changelog entry just below it is a multiple line one bullet point entry

Reply:
Sorry, I'm blind. You're right. But the next one is what I suggested :) Anyway, it's fine.

Reply:
:slightly_smiling_face: